sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_log_device_command_events_v2/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+---
+friendly_name: FxA Log Device Command Events
+description: |-
+  A subset of FxA auth server logs related to "send tab" activity.
+
+  See https://bugzilla.mozilla.org/show_bug.cgi?id=1649074
```

Comment:
I'd suggest mentioning that this contains the events from when it started running in GCP, and the date range when the GCP migration started and ended.

I'd also suggest updating the description for the v1 table to clarify that it only contains events from when it was running in AWS, and the date range when the GCP migration started and ended.

Reply:
I updated the descriptions for both v1 and v2. @sean-rose please let me know if the description provides a bit more clarity now.

Reply:
It is more clear now, thanks.  I'd suggest keeping the main summary line "_A subset of FxA auth server logs related to "send tab" activity._" as the first line though, and leaving the more detailed clarification as a secondary paragraph (kind of like a Git commit message).

Reply:
Made the tweak 🙇